underscore.js

```diff
@@ -1370,6 +1370,24 @@
 
   _.property = property;
 
+  // Generates a function for a given object that returns the passed function's return value.
+  // Accepts a method name or function literal for value.
+  // If value is a string, function assumes this string is a method name on the
+  // referenced object.
+  _.method = function(value) {
+    var isFunctionLiteral = _.isFunction(value),
+        func;
+
+    return restArgs(function(obj, args) {
+      if (obj == null) { return; }
```

Comment:
any rational why should func not be called if it is a function literal and obj is null?